cinema/gamemode/localization/pirate.lua

```diff
@@ -69,7 +69,7 @@ LANG.TheaterList_NowShowing     = "NOW SAILING"
 -- modules/scoreboard/cl_request.lua
 LANG.Request_History            = "Ledger"
 LANG.Request_Clear              = "Erase"
-LANG.Request_DeleteTooltip      = "Remove ship from ledger"
+LANG.Request_DeleteTooltip      = "Remove ship from Ledger"
 LANG.Request_PlayCount          = "%d voyage(s)" -- e.g. 10 request(s)
 LANG.Request_Url                = "Request Ship"
 LANG.Request_Url_Tooltip        = "Press to request a valid video URL.\nThe button will be red when the URL is valid"
```

Comment:
`Press to request a new ship.\nWhen the ship be ready, the button'll shine red.` or something